drafts/2018-12-04-this-week-in-rust.md

```diff
@@ -121,31 +112,36 @@ decision. Express your opinions now.
 
 * [Dec  6. Pune, IN - Rust workshop at Pune, India](https://reps.mozilla.org/e/rust-community-meetup-pune/).
 * [Dec 12. Hangzhou, CN - Rust Hangzhou](https://www.meetup.com/Rust-Hangzhou/events/256338781/).
+* [Dec 15. Chennai, IN - Rust Monthly Meetup - February](https://www.meetup.com/mad-rs/).
 
 ### Europe
 
-* [Nov 29. Copenhagen, DK - Copenhagen Rust Group - Hack Night #11](http://cph.rs/).
-* [Dec  3. Karlsruhe, DE - Rust 2018 Edition](https://www.meetup.com/Rust-Hack-Learn-Karlsruhe/events/256200841/?_xtd=gqFyqTE5MzgwNjQ5OKFwp2FuZHJvaWQ&from=ref).
 * [Dec  5. Cologne, DE - Cologne meetup](https://rust.cologne/2018/12/05/rust-2018-eve.html).
 * [Dec  6. Gouda, NL - Rust Gouda - Rust 2018 edition release party](https://www.meetup.com/Rust-Gouda/events/254877742/).
```

Comment:
This meetup was delayed until Feb the 9 if I am not wrong.

```suggestion
```

Reply:
True! I missed that one! Thanks!